designs/administrator/olm/warn-managed-resource/index.md

```diff
@@ -0,0 +1,39 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Warn when modifying managed resources
+
+A kubernetes cluster will contain many resources that are managed by another resource, which can be disorienting to users if they modify/delete the resource only to have their changes immediately undone. Some examples might be a ClusterServiceVersion managed by a Subscription, a Pod managed by a Deployment or ReplicaSet, or a Deployment, Route, Service, or Configmap managed by an Operator/Operand.
+
+Admins should be made aware that any changes they make to the resource could be undone, and that they should probably be editing the resource that manages it.
+
+This design will initially be implemented for workload-type resources that are managed by an Operand, however the design is general that if can be used for many resource relationships in the future.
```

Comment:
```suggestion
This design will initially be implemented for workload-type resources that are managed by an Operand, however, the design is general enough that it can be used for many resource relationships in the future.
```